Author: Daniel Baumann <mail@daniel-baumann.ch>
Description: Making pxelinux quiet by enabling some output only in verbose mode.

diff -Naurp syslinux.orig/core/pxelinux.asm syslinux/core/pxelinux.asm
--- syslinux.orig/core/pxelinux.asm
+++ syslinux/core/pxelinux.asm
@@ -165,11 +165,13 @@ _start1:
 ;
 ; Tell the user we got this far
 ;
+%ifdef VERBOSE_MESSAGES
 		mov si,syslinux_banner
 		call writestr_early
 
 		mov si,copyright_str
 		call writestr_early
+%endif
 
 ;
 ; do fs initialize
